fitbenchmarking/fitbenchmarking
# Fixing python version

#### Description of Work

Fixes #646 

Changes badge on Travis CI to show `python 3.6` instead of `python 3.8` 

#### Testing Instructions

1. Check that Travis CI shows  `python 3.6` 

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
